about.html

```diff
@@ -164,10 +162,10 @@ <h3 class="past-conf">See the past ICT FOR NEEDS CONFERENCE.</h3>
           src="images/Black Modern Tech Program Expo Instagram Post (1).png"
           alt=""
         />
-        <a class="white-footer" href="">ICT FOR NEEDS CONFERENCE</a>
+        <a class="white-footer" href="">Ict For Needs Conference</a>
       </div>
       <div class="white-footer">
-        <p>2023 ICT FOR NEEDS CONFERENCE</p>
+        <p>2023 Ict For Needs Conference</p>
         <p>Author : Maurice O Coker</p>
       </div>
     </footer>
```

Comment:
- [ ] Dear @coker2019, for your footer, as suggested by the previous reviewer, your design is a bit different compared with the original design. The original design contains a logo aligned to the left and 3-4 lines of text aligned to the right. besides that, your footer logo should be alongside the text not between the texts. besides that, Your design has extra large spacing around it. Please address these issues to have a better alignment with the original design.

### Your design 
![image](https://user-images.githubusercontent.com/69077061/235318455-81eff534-a1f6-4f18-94fb-c4e0d5286cfb.png)

### Original design 
![image](https://user-images.githubusercontent.com/69077061/235318463-d2f4079e-a5a5-4c02-85d6-5a7ecf5ee4ae.png)